css-pseudo-4/Overview.bs

```diff
@@ -936,6 +936,49 @@ Cascading and Per-Element Highlight Styles</h3>
   css/css-pseudo/cascade-highlight-002.html
   </wpt>
 
+<h4 id="highlight-custom-properties">
+Custom Properties in Highlight Styles</h3>
+
+  A [=custom property=] used in highlight styles uses the
+  first custom property value found in the highlight cascade.
```

Comment:
This is confusing. Just add custom properties to the list in 3.2. Styling Highlights.